server/graphql-schema.js

```diff
@@ -736,7 +711,8 @@ const graphqlSchema = gql`
     """
     input TestPlanReportInput {
         testPlanVersionId: ID!
-        testPlanTarget: TestPlanTargetInput!
+        atId: ID!
```

Comment:
Thanks for looking at this so quickly! I know this is still marked as a draft but was wondering if this shouldn't include `atVersion` and `browserVersion` as well?

Reply:
Thank YOU for looking at this so quickly 😁 So, splitting up the At / Browser and AtVersion / BrowserVersion is intentional - I put the atVersion and browserVersion fields on the TestResult type. This means that reports are locked to an AT and Browser, but they're no longer locked to an exact version. And now each TestResult can have slightly different versions. Let me know if you imagined something different maybe?